docs/source/usersguide/install.rst

```diff
@@ -371,6 +375,67 @@ if we wanted to run only the plot tests with 4 processors, we run:
 If you want to run the full test suite with different build options please
 refer to our :ref:`test suite` documentation.
 
+--------------------
+Python Prerequisites
+--------------------
+
+OpenMC's :ref:`Python API <pythonapi>` works with either Python 2.7 or Python
+3.2+. In addition to Python itself, the API relies on a number of third-party
+packages. All prerequisites can be installed using `conda
+<http://conda.pydata.org/docs/>`_ (recommended), `pip
+<https://pip.pypa.io/en/stable/>`_, or through the package manager in most Linux
+distributions.
+
+.. admonition:: Required
+   :class: error
+
+   `six <https://pythonhosted.org/six/>`_
+      The Python API works with both Python 2.7+ and 3.2+. To do so, the six
```

Comment:
To be honest, I've never tried, but 3.1 is quite old. 3.2 was released around the same time as 2.7 so some of the things that were new in 2.7 were also new in 3.2, e.g., the [argparse](https://docs.python.org/dev/whatsnew/3.2.html) module which we do use. There are probably more examples as well.